themes/default/content/docs/clouds/aws/get-started/deploy-changes.md

```diff
@@ -301,7 +301,6 @@ _, err = s3.NewBucketObject(ctx, "index.html", &s3.BucketObjectArgs{
     Acl:         pulumi.String("public-read"),
 }, pulumi.DependsOn([]pulumi.Resource{
 			publicAccessBlock,
-			ownershipControls,
```

Comment:
This line actually does need to be there, I think. I'm guessing the issue you saw was instead because of [this line](https://github.com/pulumi/pulumi-hugo/pull/4035/files#diff-a06a85181197ce448e17d363732f21ac16ccdd16ca05156e7329cb467091bcb0R279), which should be defining that variable, but is instead leaving it ignored (as `_`). If you change that line to this:

```
ownershipControls, err := s3.NewBucketOwnershipControls(ctx, "ownership-controls", &s3.BucketOwnershipControlsArgs{
``` 

... it should be good. (At least that's what our automated tests are telling me.) GitHub won't let me make that suggestion directly, though, so if you wouldn't mind confirming that and changing it up, I'd appreciate it!

Reply:
Mmm, I see, the problem I was facing was the `=` instead of `:=` First I tried to do what you mentioned but I didn't see that. Thanks for your great feedback 🙂